WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -86,13 +84,11 @@ public StatusBarViewModel(string dataDir, Network network, Config config, Hosted
 
 		public string DataDir { get; }
 
-		private Network Network { get; }
```

Comment:
This shouldn't get removed, the status bar has now the same color in different networks.

![Capture](https://user-images.githubusercontent.com/52379387/90995940-b535c100-e5bd-11ea-958f-da52ff03b86f.PNG)

Reply:
Fixed.